global-styles/github-markdown.scss

```diff
@@ -1,69 +1,9 @@
-/*
- * Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
- * SPDX-FileCopyrightText: 2020 Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
+/*!
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
```

Comment:
Is it correct to change this without giving credit to Sindre? The only changeset is the use of scss nesting and the line number fix...